.canon.yaml

```diff
@@ -0,0 +1,8 @@
+rtsp:
```

Comment:
@nicksanford is this useful for you and sean or should I add it to the git ignore?

Reply:
I don't mind it :) 

It can stay from my perspective

Reply:
Canon wrapper is useful - using `antique` containers for static build PR.

Reply:
I'll add that!